Pod/Classes/Internal/SEGSegmentIntegration.m

```diff
@@ -528,6 +532,17 @@ - (void)applicationWillTerminate
     }];
 }
 
+-(void)applicationDidFinishLaunching
+{
+    [self beginBackgroundTask];
```

Comment:
Shouldn't migration be the very first thing that happens after application startup?